src/test/regress/expected/multi_explain_0.out

```diff
@@ -747,6 +747,83 @@ t
 SELECT true AS valid FROM explain_json($$
 	SELECT avg(l_linenumber) FROM lineitem WHERE l_orderkey > 9030$$);
 t
+-- Test multi shard update
+EXPLAIN (COSTS FALSE)
+	UPDATE lineitem
+	SET l_suppkey = 12;
```

Comment:
maybe add one where we have a `WHERE partcol = 1 OR partcol = 2` to see if pruning works.

Reply:
Done